pkg/gds/store/store.go

```diff
@@ -122,6 +124,14 @@ type CertificateStore interface {
 	DeleteCertReq(id string) error
 }
 
+// ReplicaStore describes how the service interacts with and manages Peers.
+type ReplicaStore interface {
```

Comment:
In the next PR, we need to add `ReplicaStore` to the `Store` interface on line `106` (before `VersionManager`).